extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/Names.java

```diff
@@ -33,11 +38,15 @@ final class Names {
     static final DotName MAP_ENTRY = DotName.createSimple(Entry.class.getName());
     static final DotName COLLECTION = DotName.createSimple(Collection.class.getName());
     static final DotName TEMPLATE_INSTANCE = DotName.createSimple(TemplateInstance.class.getName());
+    static final DotName TEMPLATE_LOCATOR = DotName.createSimple(TemplateLocator.class.getName());
     static final DotName COMPLETION_STAGE = DotName.createSimple(CompletionStage.class.getName());
     static final DotName UNI = DotName.createSimple(Uni.class.getName());
     static final DotName LOCATION = DotName.createSimple(Location.class.getName());
+    static final DotName LOCATE = DotName.createSimple(Locate.class.getName());
+    static final DotName LOCATES = DotName.createSimple(Locates.class.getName());
     static final DotName CHECKED_TEMPLATE = DotName.createSimple(io.quarkus.qute.CheckedTemplate.class.getName());
     static final DotName TEMPLATE_ENUM = DotName.createSimple(TemplateEnum.class.getName());
+    static final DotName SINGLETON = DotName.createSimple(Singleton.class.getName());
```

Comment:
There is no need to add this constant, you can just reuse the `io.quarkus.arc.processor.DotNames.SINGLETON` or even `io.quarkus.arc.processor.BuiltinScope#SINGLETON.getName()`.

Reply:
Thank you, I corrected it.